Hunting Queries/MultipleDataSources/CobaltDNSBeacon.txt

```diff
@@ -7,18 +7,21 @@
 //
 // DataSource: Syslog, VMConnection , DnsEvents
 //
-// Tactics: #C2
+// Tactics: #CommandAndControl
 //
 (union isfuzzy=true
 (
 DnsEvents
 | where TimeGenerated >= ago(1d) 
+| where TimeGenerated <= ago(14d) 
 | where Name has "aaa.stage." or Name has "post.1"
 | extend Domain = Name, SourceIP = ClientIP , DestinationIp = IPAddresses
+| extend IPCustomEntity = IPAddresses
 ),
 (
 VMConnection
 | where TimeGenerated >= ago(1d)
+| where TimeGenerated <= ago(14d) 
```

Comment:
logic seems off here -- >= ago(1d) means in the last day.
The modified query will return no results as <=ago(14d) (more than 2 weeks ago) has no overlap with the first restriction.

Reply:
fixed